### PR TITLE
Teuchos/Panzer: add FOM in mini-em for acceptance testing

### PR DESCRIPTION
**Pull request by rppawlo**
## Motivation

MiniEM needs to print a FOM for acceptance testing.

### CODE REVIEW

**Comment by rppawlo** [comment]
Example output:
```
=================================
FOM Calculation
=================================
  Number of cells = 250000
  Time for Belos Linear Solve = 72.8201 seconds
  Number of Time Steps (one linear solve per step) = 38
  FOM ( num_cells * num_steps / solver_time / 1000) = 130.458 k-cell-steps per second 
=================================
```

**Review by cgcgcg — APPROVED** [review]

File: packages/panzer/mini-em/example/BlockPrec/main.cpp [hunk 1]
```diff
@@ -712,8 +721,28 @@ int main_(Teuchos::CommandLineProcessor &clp, int argc,char * argv[])
     auto xmlOut = stacked_timer->reportWatchrXML(test_name + ' ' + std::to_string(comm->getSize()) + " ranks", comm);
     if(xmlOut.length())
       std::cout << "\nAlso created Watchr performance report " << xmlOut << '\n';
-  } else
+
+    if ( print_fom && (comm->getRank() == 0) ) {
+      std::string fom_timer_name = "Mini-EM@Mini-EM: Total Time@Mini-EM: timestepper@Mini-EM: Advance Time Step@Stratimikos: BelosLOWS";
```
Comment (bathmatt) [inline packages/panzer/mini-em/example/BlockPrec/main.cpp:726]: Can we add a check for this 
      <Parameter name="Mesh Type" type="string" value="tet"/>
to make sure we don't use the wrong maxwell.xml which is a hex mesh?

**Comment by bathmatt** [comment]
Ok. That's great

File: packages/panzer/mini-em/example/BlockPrec/main.cpp (hunk 1 above)
Comment (bathmatt) [reply packages/panzer/mini-em/example/BlockPrec/main.cpp:726]: Ok. That's great

**Pull request merged**